wagtailio/project_styleguide/templates/patterns/includes/menu_main.html

```diff
@@ -0,0 +1,38 @@
+{% load wagtailcore_tags %}
+
+<nav class="">
+    <ul class="">
+        {% for menu_item in menus %}
+            <li class="">
+                <span>{{ menu_item.name }}</span>
+                <ul class="">
+                    {% for item in menu_item.nav_items %}
+                        {% with item.value as nav_item %}
+                            <li class="">
+                                <a href="{{ nav_item.url }}" title="{{ nav_item.text }}">
```

Comment:
This explodes because the main nav does not validate that a page or url is added to each nav item

Reply:
Thanks! URL Validation fixed in 24d3f6c